conjure_oxide/tests/integration/eprime-minion/bool/literals-to-wlit-1/config.toml

```diff
@@ -1 +1 @@
-extra_rewriter_asserts=["vector_operators_have_partially_evaluated"]
```

Comment:
I was wondering what was happening there, I'll fix it.

Reply:
I can't fire it out. I copied and pasted from the main directory and it's what I already have. Is there a way to just reject those changes?

Reply:
go to the end of the first line and hit return to insert a new empty line.

Reply:
though you should pull first, since I pushed a few things onto your branch. sorry if this causes confusion, but I wanted to make sure the trace diff issue was fixed since it's not the parser's fault.